libff/algebra/fields/fp_aux.tcc

```diff
@@ -13,143 +13,187 @@
 #ifndef FP_AUX_TCC_
 #define FP_AUX_TCC_
 
-namespace libff {
+namespace libff
+{
 
 #define STR_HELPER(x) #x
 #define STR(x) STR_HELPER(x)
 
 /* addq is faster than adcq, even if preceded by clc */
-#define ADD_FIRSTADD                            \
-    "movq    (%[B]), %%rax           \n\t"      \
+#define ADD_FIRSTADD()                                                         \
+    "movq    (%[B]), %%rax           \n\t"                                     \
     "addq    %%rax, (%[A])           \n\t"
 
-#define ADD_NEXTADD(ofs)                                \
-    "movq    " STR(ofs) "(%[B]), %%rax          \n\t"   \
-    "adcq    %%rax, " STR(ofs) "(%[A])          \n\t"
+#define ADD_NEXTADD(ofs)                                                       \
+    "movq    " STR(ofs) "(%[B]), %%rax          \n\t"                          \
+                        "adcq    %%rax, " STR(ofs) "(%[A])          \n\t"
 
-#define ADD_CMP(ofs)                                  \
-    "movq    " STR(ofs) "(%[mod]), %%rax   \n\t"      \
-    "cmpq    %%rax, " STR(ofs) "(%[A])     \n\t"      \
-    "jb      done%=              \n\t"                \
-    "ja      subtract%=          \n\t"
+#define ADD_CMP(ofs)                                                           \
+    "movq    " STR(ofs) "(%[mod]), %%rax   \n\t"                               \
+                        "cmpq    %%rax, " STR(                                 \
```

Comment:
I couldn't find a good way to stop clang format doing this.  The nuclear option would be to disable for the entire file.

Reply:
Just realised I can use `//clang-format off` ... will fix

Reply:
Should be better now - sorry about that.

Reply:
Great, thanks. Yeah, let's switch clang-format on these files :)